picasso-sample/src/main/java/com/example/picasso/SampleGalleryActivity.java

```diff
@@ -71,7 +71,7 @@ private void loadImage() {
     // Index 1 is the progress bar. Show it while we're loading the image.
     animator.setDisplayedChild(1);
 
-    Picasso.with(this).load(image).into(imageView, new EmptyCallback() {
+    Picasso.with(this).load(image).fit().into(imageView, new EmptyCallback() {
```

Comment:
Well currently you'll be distorting the aspect ratio with just `.fit()` so I'm not going to merge it unless you add `.centerInside()`.